.github/workflows/update_deps.yml

```diff
@@ -0,0 +1,26 @@
+name: Monthly dependency updates
+on:
+  schedule:
+    # First of every month
+    - cron: 0 0 1 * *
+
+jobs:
+  create_issue:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Update dependencies
+        uses: imjohnbo/issue-bot@v3.3.4
+        with:
+          title: "Update dependencies"
+          body: |
+            Update all project dependencies
+            ```
+            yarn upgrade-interactive --latest
+            ```
```

Comment:
💡 Do you have a checklist of things you do for the upgrade that we could knowledge share here? Also, it might be helpful to include commands for how you look at diffs of dependencies being upgraded, so everyones process converges for how to do this.